release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -102,6 +102,24 @@ Using `PolicyGenTemplate` resources to manage and deploy polices will be depreca
 
 For more information, see xref:../edge_computing/policygenerator_for_ztp/ztp-configuring-managed-clusters-policygenerator.adoc#ztp-configuring-managed-clusters-policygenerator[Configuring managed cluster policies by using PolicyGenerator resources].
 
+[id="ocp-4-19-edge-computing-arbiter-node_{context}"]
+==== Configuring a local arbiter node
```

Comment:
```suggestion
==== Configuring a local arbiter node (Technology Preview)
```
In the release notes, we indicate tech preview features in the heading. This might be an undocumented convention, and it is not a blocker.